apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -214,49 +214,26 @@ export class AdministratorResourcesActionBlock extends Component {
 
 export class SpecialAnnouncementActionBlock extends Component {
   static propTypes = {
-    hocLaunch: PropTypes.string,
-    hasIncompleteApplication: PropTypes.bool
+    hocLaunch: PropTypes.string
```

Comment:
Are we still using this `hocLaunch` prop for anything?  I have a vague idea that it was for when we were doing short promos in succession last year (aquatic, then dance, etc). Just double checking it's still relevant.

Reply:
I'm leaving it in for now even though it's unused, because I believe we'll be doing some similar short promos this year.